src/styles/stars.scss

```diff
@@ -0,0 +1,27 @@
+.module {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &-review {
+    font-size: $primary-font-size;
+    font-weight: $main-font-weight;
+    line-height: $secondary-line-height;
+  }
+
+  .stars {
+    display: flex;
+    margin: 0;
+
+    &-star {
+      background: url(../images/star-active.svg) no-repeat center;
+      margin-right: 4px;
+      height: 16px;
+      width: 16px;
+    }
+
+    &-star:nth-child(n+5) {
+      background-image: url(../images/star.svg);
+    }
```

Comment:
rework these styles, implement active stars modifiers
`stars--4` and then 4 active stars
--3 => 3 active etc